doc/quickstart.md

```diff
@@ -26,19 +26,12 @@
 
 ## Installation
 
-Firstly, clone this GitHub repository.
+Clone this GitHub repository.
 
 ```bash
 go get github.com/openshift/sriov-network-operator
 ```
 
-Install the Operator-SDK. The following commands will put operator-sdk to your $GOPATH/bin, please make sure that path is included in your \$PATH.
```

Comment:
Instead of removing this paragraph, I think it is better we put the link of operator-sdk installation https://sdk.operatorframework.io/docs/installation/install-operator-sdk/

Reply:
In case they don't have it installed?
`deploy-cluster` will install controller-gen and kustomize if they are missing anyway.
I can of course add it if you see it necessary though.

Reply:
Yes.
We need `operator-sdk` in the `bundle` target. Although we don't use the automatically generated bundle at this moment, we plan to.

Reply:
@pliurh Alright, added the operator-sdk install link as requested